services/user.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+
+
+User = get_user_model()
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None,
+) -> User:
+    user = User.objects.create_user(username=username)
```

Comment:
This line of code does not meet the requirement 'Don't Repeat Yourself' from the checklist. The `save()` method is called in the `create_user` method of the User model and you call it again on line 23. You can avoid this by creating the user instance without saving it to the database using `User(username=username)` and then save it after all changes on line 23.